### PR TITLE
Core/Opcodes: updated values of few opcodes

### PR DESCRIPTION
**Pull request by Goatform**
- SMSG_LOOT_LIST
- SMSG_LOOT_RELEASE_RESPONSE
- SMSG_PET_ACTION_SOUND
- SMSG_PLAYERBOUND
- SMSG_PLAY_MUSIC
- SMSG_PLAY_OBJECT_SOUND
- SMSG_PVP_CREDIT
- SMSG_PVP_OPTIONS_ENABLED
- SMSG_SPELLORDAMAGE_IMMUNE
- SMSG_SPELL_COOLDOWN

### CODE REVIEW

**Comment by DDuarte** [comment]
Please add these opcodes to WPP (https://github.com/TrinityCore/WowPacketParser/blob/master/WowPacketParser/Enums/Version/V6_0_3_19103/Opcodes.cs) if they do not exist there already.

Later on, we'll take the opcodes from WPP and sync it with TC (e.g 36e50ff8ae48855e8dd240aa602058b1593400a8)

File: src/server/game/Server/Protocol/Opcodes.h [hunk 1]
```diff
@@ -1347,10 +1347,10 @@ enum OpcodeServer : uint32
     SMSG_SPELLLOGEXECUTE                              = 0x1823,
     SMSG_SPELLLOGMISS                                 = 0xBADD,
     SMSG_SPELLNONMELEEDAMAGELOG                       = 0x11FB,
-    SMSG_SPELLORDAMAGE_IMMUNE                         = 0xBADD,
+    SMSG_SPELLORDAMAGE_IMMUNE                         = 0x1313,
```
Comment (DDuarte) [inline src/server/game/Server/Protocol/Opcodes.h]: Missing in WPP

File: src/server/game/Server/Protocol/Opcodes.h [hunk 2]
```diff
@@ -1228,9 +1228,9 @@ enum OpcodeServer : uint32
     SMSG_PROCRESIST                                   = 0xBADD,
     SMSG_PROPOSE_LEVEL_GRANT                          = 0xBADD,
     SMSG_PUREMOUNT_CANCELLED_OBSOLETE                 = 0xBADD,
-    SMSG_PVP_CREDIT                                   = 0xBADD,
+    SMSG_PVP_CREDIT                                   = 0x13A9,
```
Comment (DDuarte) [inline src/server/game/Server/Protocol/Opcodes.h]: Missing in WPP

File: src/server/game/Server/Protocol/Opcodes.h [hunk 3]
```diff
@@ -1209,14 +1209,14 @@ enum OpcodeServer : uint32
     SMSG_PET_UPDATE_COMBO_POINTS                      = 0xBADD,
     SMSG_PLAYED_TIME                                  = 0x038B,
     SMSG_PLAYERBINDERROR                              = 0xBADD,
-    SMSG_PLAYERBOUND                                  = 0xBADD,
+    SMSG_PLAYERBOUND                                  = 0x078E,
```
Comment (DDuarte) [inline src/server/game/Server/Protocol/Opcodes.h]: Missing in WPP

File: src/server/game/Server/Protocol/Opcodes.h [hunk 4]
```diff
@@ -1085,10 +1085,10 @@ enum OpcodeServer : uint32
     SMSG_LOOT_CLEAR_MONEY                             = 0xBADD,
     SMSG_LOOT_CONTENTS                                = 0xBADD,
     SMSG_LOOT_ITEM_NOTIFY                             = 0xBADD,
-    SMSG_LOOT_LIST                                    = 0xBADD,
+    SMSG_LOOT_LIST                                    = 0x03A4,
     SMSG_LOOT_MASTER_LIST                             = 0xBADD,
     SMSG_LOOT_MONEY_NOTIFY                            = 0x12A9,
-    SMSG_LOOT_RELEASE_RESPONSE                        = 0xBADD,
+    SMSG_LOOT_RELEASE_RESPONSE                        = 0x0BD3,
```
Comment (DDuarte) [inline src/server/game/Server/Protocol/Opcodes.h]: Missing in WPP

**Comment by Goatform** [comment]
Okay

**Comment by DDuarte** [comment]
They were added to WPP in https://github.com/TrinityCore/WowPacketParser/commit/0ce12534a7b3ef39fe28f4754682bb1293a5501b
Thanks for them, @Goatform !

**Comment by Goatform** [comment]
You're welcome!

**Pull request closed without merging**